Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM jupyter/datascience-notebook:lab-2.2.9
```

Comment:
I wonder if it's better to start from something leaner, since this also includes R?

Reply:
Indeed, I selected this because it's the first in the stack that also includes Julia. Would you go for `jupyter/scipy-notebook` or even smaller?

Reply:
![image](https://user-images.githubusercontent.com/8681967/104118172-b6c71200-532f-11eb-9ff8-8d5c5fd4c9af.png)
That has pluto but not IJulia. Guess that's fine, so I'm uploading updated!!